config.webpack.js

```diff
@@ -26,6 +26,9 @@ const defineEnvPlugin = new webpack.DefinePlugin({
   __RX_ENABLED__: JSON.stringify(process.env.RX_ENABLED || false),
   __PENDO_ENABLED__: JSON.stringify(process.env.PENDO_ENABLED || true),
   __CLINICS_ENABLED__: JSON.stringify(process.env.CLINICS_ENABLED || false),
+  __KEYCLOAK_URL__: JSON.stringify(process.env.KEYCLOAK_URL || null),
```

Comment:
We will need to update the `Dockerfile` to accept these as build arguments in order to run keycloak on remote docker builds.

As well, when the time comes to test these on the remotes, we'll need to update the `artifact.sh` script in the `tools` repo to pass them in from Travis.  Should that be done now?

Reply:
`Dockerfile` not being my forte, I'll rely on you to let me know if I've updated it properly. Also added a PR for the `tools` repo at: https://github.com/tidepool-org/tools/pull/92 and have tagged you for review on it (`bash` script also not being something really in my wheelhouse).